system/datatypes/peer_reviewed_article.php

```diff
@@ -718,19 +735,85 @@ public function commentConfirmAction(HyphaRequest $request) {
 			return null;
 		}
 
-		$pending = (bool)$comment->getAttribute(self::FIELD_NAME_DISCUSSION_COMMENT_PENDING);
-		if ($pending) {
+		$pending = [];
```

Comment:
I think pending is no longer a good variable name here. Something like `$confirmed` is better, since it indicates that something was confirmed (even though that does mean it used to be pending). Also, I think that simply splitting this into two variables (e.g. `$confirmedComment` and `$confirmedSubscription`) is more clear.

Finally: Is there ever a case where you can confirm a subscription without confirming a comment? Probably not in practice, but maybe it is enough to track this separately just for abnormal circumstances?

Reply:
fixed